<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1924</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1981</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="backgroundL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1980</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/bg/img/images.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lowerL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>770</y>
       <width>1921</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/bg/img/floor .gif</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="upperL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1381</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/img/img/init.gif</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="statusL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>491</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string> ...........</string>
     </property>
    </widget>
    <widget class="QLabel" name="botL">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>0</y>
       <width>531</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/bg/img/bot.gif</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="exitB">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1000</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/bg/img/exit - Copy.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="novaL">
     <property name="geometry">
      <rect>
       <x>1380</x>
       <y>600</y>
       <width>541</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>HP Simplified Hans</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Nova :- </string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="bgL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>771</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/bg/img/animation.gif</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>690</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="userL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>621</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>User :-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>backgroundL</zorder>
    <zorder>lowerL</zorder>
    <zorder>bgL</zorder>
    <zorder>upperL</zorder>
    <zorder>statusL</zorder>
    <zorder>novaL</zorder>
    <zorder>exitB</zorder>
    <zorder>botL</zorder>
    <zorder>label</zorder>
    <zorder>userL</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
